api-spec/api-spec.md

```diff
@@ -38,22 +38,21 @@ STAC API.
 
 ## HTTP Request Methods and Content Types
 
-The OAFeat and STAC APIs follow a RESTful model. A core principal of this is the use of HTTP Request Methods (verbs) and
-the `Content-Type` header to drive behavior. This section describes how these are used in the OAFeat and STAC endpoints. 
+The OAFeat and STAC APIs follow a RESTful model. A core principal of this is the use of HTTP Request Methods ("verbs") and
+the `Content-Type` header to drive behavior on resources ("nouns"). This section describes how these are used in the OAFeat and STAC endpoints. 
 
-1. **Required** GET (both OAFeat and STAC)
-2. **Recommended** POST `Content-Type: application/x-www-form-urlencoded` with the corresponding content body format.
-3. **Recommended** POST `Content-Type: multipart/form-data` with the corresponding content body format.
-4. **Optional** **STAC endpoint /search only** POST `Content-Type: application/json`, where the content body is a JSON 
+1. **Required** Both OAFeat and STAC: GET using query string parameters
+1. **Recommended** Only STAC: POST `Content-Type: application/x-www-form-urlencoded` with the corresponding content body format equivalent to the parameters defined for GET.
```

Comment:
The list is strange... (1, 1, 1, ...?)

Also, I'm still not sure what it means. Does this translate to: "All POST endpoints are recommended to implement this content type" (and does it include the Transaction extension)? Or is this only valid for POST /search?

Reply:
Markdown doesn't care what number you actually put there, it just generates a sequence, so I tend to just use 1. so I don't have to constantly renumber them when modifying the order.

Reply:
This is specifically referring to `/search`, as that is the only endpoint part of the base STAC API. 

This would not apply to other API extensions that have POST endpoints, such as Transactions, as they don't also have`GET + query string` defined. Also, IIRC, all of the Transactions endpoints are just the OAFeat ones with POST + JSON applied. I can clarify this. 

Maybe we should also **recommend** that the OAFeat GET endpoints also have `POST form-urlencoded` implemented?

Reply:
I'd be cool to recommend that.